src/core/api/rtd_deposited_file_check/azureblob_policy.xml

```diff
@@ -4,34 +4,10 @@
     <inbound>
       <base />
       <authentication-managed-identity resource="https://storage.azure.com/" output-token-variable-name="msi-access-token" ignore-error="false" />
-
```

Comment:
Do you think is better to put a comment with the issue, to simplify the futures updates?